docs/manual/config/config_file_reference.rst

```diff
@@ -21,7 +21,7 @@ CycloneDDS configuration
 //CycloneDDS/Domain
 *******************
 
-Attributes: [Id](`//CycloneDDS/Domain[@Id]`_)
```

Comment:
This is definitely much better, but I wonder if it would be possible to show `Id` the reader but have it be a link target for the definition of the attribute (as in `<a href="...">Id</a>`): that would make it read like:

Attributes: Id
Children: Compatibility, Discovery, General, ...

which in my opinion would look much nicer.

(The same goes for the child elements, of course.)

Reply:
@eboasson i agree and fixed it